src/main/java/no/sikt/sws/WorkspaceStripper.java

```diff
@@ -29,16 +30,20 @@ public static String remove(String body, String workspace) {
     }
 
     public static String prefixUrl(String index, String workspace) {
+        logger.info("prefixing " + workspace);
         if (index == null) {
-            return null;
+            return workspace + "-*";
+
+        } else if (index.startsWith("_")) {
+            return index;
```

Comment:
As we dont allow _ actions after Fredriks changes today - is this check needed?

Reply:
If you remove this then the whole method could be simplified:



var affix = (index == null) ? "*" : index
return workspace + "-" + affix